### PR TITLE
Update dotnet-install topic

### LINKED ISSUE #5736: Provide dotnet-install for .net core 2.1
# Issue Title

The existing dotnet-install.* scripts appear to be out of date. 
Can you:
- update them for the latest release?
- provide documentation how to use the updated scripts?

Best regards

### LINKED ISSUE #7410: Documentation for flags needs to be updated
In particular the script now says that --shared-runtime is deprecated and to use --runtime instead

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a2e484a9-5e0c-eb3e-1413-7b507c4fa07d
* Version Independent ID: 5bc7caa5-f1c2-d7d5-fe02-80c65cca9672
* Content: [dotnet-install scripts](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script)
* Content Source: [docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-install-script.md)
* Product: **dotnet-core**
* GitHub Login: @blackdwarf
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #7410 
Fixes #5736 

PR #6736  should be merged first

/cc @johnbeisner @KathleenDollard

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: docs/core/tools/dotnet-install-script.md [hunk 1]
```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
...
+The default value is `LTS`. For more information on .NET support channels, see the [.NET Support Policy](https://www.microsoft.com/net/platform/support-policy#dotnet-core) page.
 
 `-Version <VERSION>`
 
 Represents a specific build version. The possible values are:
 
-- `latest` - Latest build on the channel (used with the `-Channel` option)
-- `coherent` - Latest coherent build on the channel; uses the latest stable package combination (used with Branch name `-Channel` options)
-- Three-part version in X.Y.Z format representing a specific build version; supersedes the `-Channel` option. For example: `2.0.0-preview2-006120`
+* `latest` - Latest build on the channel (used with the `-Channel` option).
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-install-script.md]: I think that I'd consider indenting everything belonging to a version. Currently, the options don't really stand out from their related text.

File: docs/core/tools/dotnet-install-script.md [hunk 2]
```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
...
+* `coherent` - Latest coherent build on the channel; uses the latest stable package combination (used with Branch name `-Channel` options).
+* Three-part version in X.Y.Z format representing a specific build version; supersedes the `-Channel` option. For example: `2.0.0-preview2-006120`.
 
-If omitted, `-Version` defaults to `latest`.
+If not specified, `-Version` defaults to `latest`.
 
 `-InstallDir <DIRECTORY>`
 
-Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\.dotnet*. Note that binaries are placed directly in the directory.
+Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\.dotnet*. Binaries are placed directly in the directory.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-install-script.md]: nit: the directory --> this directory

File: docs/core/tools/dotnet-install-script.md [hunk 3]
```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
...
+If set, the installation folder isn't exported to the path for the current session. By default, the script modifies the PATH, which makes the CLI tools available immediately after install.
+
+`-Verbose`
+
+Displays diagnostics information.
 
 `-AzureFeed`
 
-Specifies the URL for the Azure feed to the installer. It isn't recommended that you change this value. The default is `https://dotnetcli.azureedge.net/dotnet`.
+Specifies the URL for the Azure feed to the installer. It isn't recommended that you change this value. The default value is `https://dotnetcli.azureedge.net/dotnet`.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-install-script.md]: nit: It isn't recommended that you change --> We recommend that you don't change

File: docs/core/tools/dotnet-install-script.md [hunk 4]
```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
...
+
+`-UncachedFeed`
+
+It allows changing the URL for the Uncached feed used by this installer. It isn't recommended that you change this value.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-install-script.md]: It allows --> Allows
nit: It isn't recommended that you --> We recommend that you don't

File: docs/core/tools/dotnet-install-script.md [hunk 5]
```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
...
+
+`-NoCdn`
+
+Disables downloading from the Azure CDN and uses the uncached feed directly.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-install-script.md]: should you say what CDN is?

File: docs/core/tools/dotnet-install-script.md [hunk 6]
```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
...
+`ProxyUseDefaultCredentials`
 
-If set, the installer uses the credentials of the current user to access the proxy server that is specified by the `-Proxy` parameter. (Only valid for Windows)
+If set, the installer uses the credentials of the current user when using proxy address. (Only valid for Windows)
 
-`--verbose`
+`-SkipNonVersionedFiles`
 
-Display diagnostics information.
+Skips installing non-versioned files if they already exist, such as *dotnet.exe*.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-install-script.md]: non-versioned files, such as \*dotnet.exe\* if they already exist.

**Review by rpetrusha — APPROVED** [review]
This looks good, @mairaw. I've left a few comments and suggestions for you to consider before you merge.

File: docs/core/tools/dotnet-install-script.md (hunk 1 above)
Comment (mairaw) [reply docs/core/tools/dotnet-install-script.md]: I'm experimenting using bold for the options like the Azure CLI docs style. I'll see if that helps. If it does, I'll apply the same change to the other CLI commands on a separate PR.

**Pull request merged**